Assets/Scripts/Input/InputSystem.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using UnityEngine.AddressableAssets;
+using UnityEngine.InputSystem;
+
+namespace Input 
```

Comment:
Namespaces start with the project name and than the folder.

```suggestion
namespace UntitledCube.Input 
```

Reply:
Changed the namespace